### DISCUSSION TITLE
[Feature] Map view of an album

### DISCUSSION THREAD

**Discussion by qrlt** [immich-app/immich#8504] [Feature Request]
At the moment, there is no possibility to get a map view of the photos in an album.
For ex, if albums are used for different travel trips, it would be very useful to see all the location of the photos taken from a single album.

Acceptance criteria:

- [ ] From the album view, there would be a possibility to open map view and only the photos in the album would be displayed on the map

### Platform
- [ ]     Server
- [x]     Web
- [x]     Mobile

**Comment by bo0tzz** [comment]
Duplicate of #3418
